src/components/UserProfile/getTitanTokenCount.ts

```diff
@@ -0,0 +1,16 @@
+import { GetTokenHoldersJSON } from "../Web3/GraphQueries";
+import { fetchJson, getCidImage } from "../Web3/Ipfs";
+import titanTokenPlaceholder from "../../assets/images/placeholders/titan-token-placeholder.png";
+
+export const getTitanTokenCount = async (accountAddress: string) => {
+  const tokenHolders = await GetTokenHoldersJSON();
+  const userBalanceData = tokenHolders.data.users.find((user) => user.address === accountAddress.toLowerCase());
```

Comment:
maybe add an if statement checking if tokeholders isn't empty to avoid stuff breaking when GetTokenHolders function doesn't work